node/src/main/scala/coop/rchain/node/api/WebApi.scala

```diff
@@ -36,7 +37,9 @@ trait WebApi[F[_]] {
   def deploy(request: DeployRequest): F[String]
 
   // Read data (listen)
-  def listenForDataAtName(request: DataRequest): F[DataResponse]
+  def listenForDataAtName(request: DataAtNameRequest): F[DataAtNameResponse]
+
+  def listenForDataAtPar(request: DataAtParRequest): F[DataAtParResponse]
```

Comment:
Let's just call it `getDataAtPar` as in BlockAPI to easily distinguish from old method (which is misleading with prefix _listen_).

Reply:
Done in [commit](https://github.com/rchain/rchain/pull/3584/commits/02edbaa462e7ee0f1c1dda1200e95163be4497d6).